### ISSUE TITLE
Chained methods aren't banned properly

### ISSUE THREAD

**Issue by marines** [palantir/tslint#1234]
### Bug Report

- __TSLint version__: `3.9`
- __TypeScript version__: `1.8.10`
- __Running TSLint via__: `gulp-tslint`

#### TypeScript code being linted

```ts
_.keys(obj).forEach((key: string) => console.log(key));
```

with `tslint.json`:

```json
{
  "rules": {
    "ban": [true,
        [ "_", "forEach" ],
        [ "_", "each" ],
        [ "$", "each" ],
        [ "angular", "forEach" ]
    ]
}
```

#### Actual behavior

`function invocation disallowed: _.forEach`

#### Expected behavior

It should pass since `forEach` is native Array method (`_.keys()` doesn't even return `_`).

Splitting it to:

```javascript
var keys: string[] = _.keys(obj);
keys.forEach((key: string) => console.log(key));
```

passes linting.
